src/Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing/Configuration/MetricsCapturingModuleConfigurationParameters.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing.Configuration
+{
+	/// <summary>
+	/// Encapsulates configuration parameters used by Functional.CQS.AOP.IoC.CastleWindsor.MetricsCapturing components.
```

Comment:
Should this say `SimpleInjector` instead of `CastleWindsor`?

Reply:
Yep!